content/en/blog/_posts/2025-04-23-jobs-backofflimitperindex-goes-ga.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: blog
+title: "Kubernetes 1.33: Job's Backoff Limit Per Index Goes GA"
+date: 2025-04-23
+slug: kubernetes-1-33-jobs-backoff-limit-per-index-goes-ga
+author: >
+  [Michał Woźniak](https://github.com/mimowo) (Google)
+---
+
+In Kubernetes v1.33, the _Backoff Limit Per Index_ feature reaches general
+availability (GA). This blog describes the Backoff Limit Per Index feature and
+its benefits.
+
+## About Backoff Limit Per Index
+
+When you run workloads on Kubernetes, you must consider scenarios where Pod
+failures can affect the completion of your workloads. Ideally, your workload
+should tolerate transient failures and continue running.
+
+To achieve failure tolerance in Kubernetes Job, you can set the
+`spec.backoffLimit` field. This field specifies the total number of tolerated
+failures.
+
+However, for workloads where every index is considered independent, like
+[embarassingly parallel](https://en.wikipedia.org/wiki/Embarrassingly_parallel)
+workloads, the `spec.backoffLimit` field is often not flexible enough if you use
+Indexed Jobs. For example, integration tests. In that scenario, a fast-failing
```

Comment:
Before you go into the details of the problem, can you explain what an index/indexed job is in this context? Some short explanation and maybe some links would be great

Reply:
Thanks, I have rephrased the section to make it a bit more clear, and added a link to the relevant guide on using Indexed Jobs for parallel workloads with static work assignments. 

Let me know what you think, I'm happy to adjust it more.

Reply:
LGTM, thanks!